src/lib/ActionPrototype.js

```diff
@@ -1,29 +1,57 @@
-var fetcher = require('../lib/jsonFetcher');
-var AppDispatcher = require('../lib/ep-dispatcher');
-var KeyMirror = require('keymirror');
+import fetcher from '../lib/jsonFetcher'
```

Comment:
@BJK Again, just curious: why is `fetcher` the only thing imported here that is not CamelCased?